src/utilities/CatalogApiProcessed/CatalogApiProcessed.ts

```diff
@@ -158,14 +160,16 @@ export class CatalogApiProcessing {
                             tableElement: tableElement
                         }
                     };
-                    resources.set(name, res);
+                    if (name !== null) {
```

Comment:
Maybe we can use isFinite?

Reply:
It's a string, so we can't use isFinite.